plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaFilterManager.java

```diff
@@ -140,10 +142,17 @@ public KafkaFilteringResult getKafkaFilterResult(
 
             // push down partitions
             final Set<Long> finalPartitionIdsFiltered = partitionIdsFiltered;
-            List<PartitionInfo> partitionFilteredInfos = partitionInfos.stream()
+            partitionInfos = partitionInfos.stream()
```

Comment:
Why this change ?

Reply:
This change is to reuse the following，like line:157